adagrams/game.py

```diff
@@ -1,11 +1,174 @@
+import random
+
+LETTER_POOL_W_VALUES = {
+    'A': {
+        'qty': 9, 
+        'value': 1
+    },
+    'B': {
+        'qty': 2,
+        'value': 3
+    },
+    'C': {
+        'qty': 2, 
+        'value': 3
+    },
+    'D': {
+        'qty': 4, 
+        'value': 2
+    },
+    'E': {
+        'qty': 12,
+        'value': 1
+    },
+    'F': {
+        'qty': 2, 
+        'value': 4
+    },
+    'G': {
+        'qty': 3, 
+        'value': 2
+    },
+    'H': {
+        'qty': 2, 
+        'value': 4
+    },
+    'I': {
+        'qty': 9,
+        'value' :1
+    },
+    'J': {
+        'qty': 1, 
+        'value': 8
+    },
+    'K': {
+        'qty': 1, 
+        'value': 5
+    },
+    'L': {
+        'qty': 4, 
+        'value': 1
+    },
+    'M': {
+        'qty': 2, 
+        'value': 3
+    },
+    'N': {
+        'qty': 6, 
+        'value': 1
+    },
+    'O': {
+        'qty': 8,
+        'value': 1
+    },
+    'P': {
+        'qty': 2, 
+        'value': 3
+    },
+    'Q': {
+        'qty': 1, 
+        'value': 10
+    },
+    'R': {
+        'qty': 6,
+        'value': 1
+    },
+    'S': {
+        'qty': 4, 
+        'value': 1
+    },
+    'T': {
+        'qty': 6, 
+        'value': 1
+    },
+    'U': {
+        'qty': 4, 
+        'value': 1
+    },
+    'V': {
+        'qty': 2, 
+        'value': 4
+    },
+    'W': {
+        'qty': 2, 
+        'value': 4
+    },
+    'X': {
+        'qty': 1, 
+        'value': 8
+    },
+    'Y': {
+        'qty': 2, 
+        'value': 4
+    },
+    'Z': {
+        'qty': 1,
+        'value': 10
+    }
+}
+
 def draw_letters():
-    pass
+    letters = []
+    frequency_of_letters = []
+    for letter, frequency in LETTER_POOL_W_VALUES.items():
+        letters.append(letter)
+        frequency_of_letters.append(frequency['qty'])
+    # random.sample returns a new list containing elements from the population
```

Comment:
Interesting approach! I appreciate that you added the comment about how the `random.sample` function works. It's a good idea, at this stage, to include comments like this when using functions from imported modules we haven't covered in the curriculum. 

Expanding on that, it's wise to remember that not every language has the same built-in functions, so we should be able to implement this same functionality ourselves. In the projects, we don't ask for any features that we don't think you should be able to write yourself. For drawing a random hand, the only external function that would be "required" is a way to pick a random number (such as randint). At this stage in our coding journeys, it's often more valuable to re-implement things by hand to practice thinking about how to break down and approach the problem at hand, than to use a single library function to solve the problem for us.